core/block_drag_surface.js

```diff
@@ -209,6 +222,16 @@ Blockly.BlockDragSurfaceSvg.prototype.getCurrentBlock = function() {
   return /** @type {Element} */ (this.dragGroup_.firstChild);
 };
 
+/**
+ * Gets the translation of the child block surface
+ * This surface is in charge of keeping track of how much the workspace has
+ * moved.
+ * @return {!Blockly.utils.Coordinate} The amount the workspace has been moved.
+ */
+Blockly.BlockDragSurfaceSvg.prototype.getWSTranslation = function() {
```

Comment:
I think in other places where we abbreviate workspace, we do it as `Ws` e.g. `Blockly.Mutator.findParentWs`, `getWsCoordinate`, etc.

Reply:
Yup, good catch. Done.